addons/account/models/account_move.py

```diff
@@ -504,46 +2075,555 @@ def _run_post_draft_to_post(self):
+        help="Technical field holding the debit - credit in order to open meaningful graph views from reports")
+    amount_currency = fields.Monetary(string='Balance in Currency', store=True, copy=True,
+        help="The amount expressed in an optional other currency if it is a multi-currency entry.")
+    price_subtotal = fields.Monetary(string='Subtotal', store=True, readonly=True,
+        currency_field='always_set_currency_id')
+    price_total = fields.Monetary(string='Total', store=True, readonly=True,
+        currency_field='always_set_currency_id')
+    reconciled = fields.Boolean(compute='_amount_residual', store=True)
+    blocked = fields.Boolean(string='No Follow-up', default=False,
+        help="You can check this box to mark this journal item as a litigation with the associated partner")
+    date_maturity = fields.Date(string='Due date', index=True,
+        help="This field is used for payable and receivable journal entries. You can put the limit date for the payment of this line.")
+    currency_id = fields.Many2one('res.currency', string='Currency')
+    partner_id = fields.Many2one('res.partner', string='Partner', ondelete='restrict')
+    product_uom_id = fields.Many2one('uom.uom', string='Unit of Measure')
+    product_id = fields.Many2one('product.product', string='Product')
+
+    # ==== Origin fields ====
+    reconcile_model_id = fields.Many2one('account.reconcile.model', string="Reconciliation Model", copy=False)
+    payment_id = fields.Many2one('account.payment', string="Originator Payment", copy=False,
+        help="Payment that created this entry")
+    statement_line_id = fields.Many2one('account.bank.statement.line',
+        string='Bank statement line reconciled with this entry',
+        index=True, copy=False, readonly=True)
+    statement_id = fields.Many2one(related='statement_line_id.statement_id', store=True, index=True, copy=False,
+        help="The bank statement used for bank reconciliation")
+
+    # ==== Tax fields ====
+    tax_ids = fields.Many2many('account.tax', string='Taxes')
+    tax_line_id = fields.Many2one('account.tax', string='Originator tax', ondelete='restrict', store=True,
+        compute='_compute_tax_line_id')
+    tax_base_amount = fields.Monetary(string="Base Amount", store=True,
+        currency_field='company_currency_id')
+    tax_exigible = fields.Boolean(string='Appears in VAT report', default=True,
+        help="Technical field used to mark a tax line as exigible in the vat report or not (only exigible journal items"
+             " are displayed). By default all new journal items are directly exigible, but with the feature cash_basis"
+             " on taxes, some will become exigible only when the payment is recorded.")
+    tag_ids = fields.Many2many(string="Tags", comodel_name='account.account.tag', ondelete='restrict',
+        help="Tags assigned to this line by the tax creating it, if any. It determines its impact on financial reports.")
```

Comment:
this needs to be renamed into tax_tag_ids, probably (cc @oco-odoo )

Reply:
will be done later